### PR TITLE
rename SimpleSAMLphp directories

### PR DESCRIPTION
**Pull request by briskt**
### Changed (breaking)
- Rename all `lib` directories to `src`
- Rename all `www` directories to `public`

To keep this pull request relatively small in scope, tests have been skipped because they are not yet passing.

[Reference](https://simplesamlphp.org/docs/stable/simplesamlphp-upgrade-notes-2.0.html#changes-relevant-for-module-developers)

https://itse.youtrack.cloud/issue/IDP-869

### CODE REVIEW

**Comment by jason-jackson** [comment]
Should vendor files be changing too?

**Comment by briskt** [comment]
> Should vendor files be changing too?

Vendor files aren't committed to the repo

File: dockerbuild/run.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ set -x
 set -e
 
 # This is a temporary fix (bug workaround) until ssp 2.0 is in use
-sed -i 's_\(\\SimpleSAML\\Error\\Assertion::installHandler()\)_// \1 _' /data/vendor/simplesamlphp/simplesamlphp/www/_include.php
+sed -i 's_\(\\SimpleSAML\\Error\\Assertion::installHandler()\)_// \1 _' /data/vendor/simplesamlphp/simplesamlphp/public/_include.php
```
Comment (jason-jackson) [inline dockerbuild/run.sh:10]: Sorry, I'll put the comment here to show what I mean. The vendor reference is changing from `www` to `public` I assume that's fine, but wanted to double-check.

Comment (briskt) [reply dockerbuild/run.sh:10]: Yes, that's correct.

**Pull request merged**